ginivision/src/main/java/net/gini/android/vision/internal/camera/api/CameraController.java

```diff
@@ -335,45 +336,56 @@ private void configureCamera(Activity activity) {
         }
 
         Camera.Parameters params = mCamera.getParameters();
+        selectPictureSize(params);
+        selectPreviewSize(params);
+        selectFocusMode(params);
+        selectFlashMode(params);
```

Comment:
Good point. I thought about this and what I could propose is something like:
```
// The CameraConfiguration would be an interface, this would allow to change the configuration in it's implementation
newParams = cameraConfiguration.apply(params);
mCamera.setParameters(newParams);
```
In the `apply()` method we can then change the parameters as we see fit.